tools/respecDocWriter.js

```diff
@@ -96,6 +96,19 @@ const tasks = {
         const msg = `HTTP Error ${response.code}: ${url}`;
         throw new Error(msg);
       }
+      const isRespecDoc = yield nightmare
+        .wait(function(){
+          return document.readyState === "complete";
+        }) 
+        .evaluate(function(){
+          const selector = "script[src='https://www.w3.org/Tools/respec/respec-w3c-common']";
```

Comment:
I'll add that as first check. Problem is that I've seen people add ReSpec asycn. I'll add as first check:

```
          if(document.hasOwnProperty("respectIsReady")){
            return true;
          }
```